frontend/public/components/cluster-overview.jsx

```diff
@@ -151,20 +146,21 @@ const LimitedGraphs = ({openshiftFlag}) => {
   </div>;
 };
 
-const GraphsPage = ({fake, limited, namespace, openshiftFlag}) => {
-  if (flagPending(openshiftFlag)) {
+const GraphsPage = connectToFlags(FLAGS.OPENSHIFT)(({flags, mock, limited, namespace}) => {
+  if (flagPending(flags.OPENSHIFT)) {
```

Comment:
This is not an intuitive API, but it should be.

```suggestion
  const openshiftFlag = flags[FLAGS.OPENSHIFT];
  if (flagPending(openshiftFlag)) {
```